src/main/java/lotto/Controller/LottoController.java

```diff
@@ -1,26 +1,46 @@
 package lotto.Controller;
 
+import lotto.constant.LottoMessage;
 import lotto.domain.*;
 import lotto.utils.LottoUtils;
 import lotto.view.LottoInputView;
 import lotto.view.LottoOutPutView;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class LottoController {
 
     public static void start() {
         Money money = new Money(LottoUtils.StringToInt(LottoInputView.readMoney()));
-        Lottos lottos = Lottos.buyLottos(money.lottoCount());
-        LottoOutPutView.writeBuyLottos(lottos);
+        int manualLottoCount = LottoUtils.StringToIntNum(LottoInputView.readManualLottoCount());
+        Lottos manualLottos = new Lottos(new ArrayList<>());
+        if (manualLottoCount > money.lottoCount()) {
+            throw new IllegalArgumentException(LottoMessage.ERROR_MESSAGE_MANUAL_LOTTO_COUNT);
+        }
+        if (manualLottoCount > 0) {
+            manualLottos = readBuyManualLottos(manualLottoCount, LottoInputView.readManualLotto(manualLottoCount));
+        }
+        Lottos autoLottos = Lottos.buyAutoLottos(money.lottoCount()-manualLottoCount);
+        Lottos totalLottos = Lottos.mergeLottos(manualLottos, autoLottos);
+        LottoOutPutView.writeBuyLottos(manualLottos, autoLottos);
         Lotto winLotto = readWinLotto(LottoInputView.readLottoWinNumber());
-        int bonusNumber = LottoUtils.StringToInt(LottoInputView.readBonusNumber());
+        int bonusNumber = LottoUtils.StringToIntNum(LottoInputView.readBonusNumber());
         LottoNumber bonus = new LottoNumber(bonusNumber);
         winLotto.duplicateWinBonus(bonus);
-        LottoResult result = new LottoResult(lottos, winLotto, money.totalLottoPrice(), bonus);
+        LottoResult result = new LottoResult(totalLottos, winLotto, money.totalLottoPrice(), bonus);
         LottoOutPutView.writeWinResult(result);
     }
 
+    private static Lottos readBuyManualLottos(int manualLottoCount, List<String> readManualLotto) {
+        List<Lotto> manualLottos = new ArrayList<>();
+        for (int i  = 0; i < manualLottoCount; i++){
+            List<Integer> manualLotto = LottoUtils.stringToLottoNumbers(readManualLotto.get(i));
+            manualLottos.add(new Lotto(manualLotto));
+        }
+        return new Lottos(manualLottos);
```

Comment:
`manualLottoCount`는 필요없을것 같아요!
그리고
```suggestion
        return new Lottos(readManualLotto.stream()
                .map(LottoUtils::stringToLottoNumbers)
                .map(Lotto::new)
                .collect(Collectors.toList()));
```
이렇게 해도 될거같네요!😀

Reply:
👍👏👍👏👍👍👍 이렇게 하면 manualLottoCount가 필요 없네요!! 자세히 알려주셔서 감사합니다👍